### DISCUSSION TITLE
Fief behind MTLS

### DISCUSSION THREAD

**Discussion by AlissonMachadoEmbitel** [fief-dev/fief#396] [Q&A]
Hello Guys,

I am trying to implement the fief server behind a mtls server. 
we are following this documentation https://docs.fief.dev/integrate/javascript/backend/express/

however i don't see how i can add the client certificate.

**Comment by fief-bailiff** [comment]
Hail, @AlissonMachadoEmbitel 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by frankie567** [comment]
Hi @AlissonMachadoEmbitel 👋 

Unfortunately, our client does not allow to pass advanced config that would allow MTLS. I'll add it to the backlog!
